quiz.rb

```diff
@@ -2,19 +2,57 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+  attr_reader :name, :breed, :age
```

Comment:
One small point - it's convention that `attr_whatever` go before the `initialize` method.